docs/testing/ie-11-testing.md

```diff
@@ -0,0 +1,44 @@
+---
+ms.date: 05/14/2020
+description: 'Test your add-in using Internet Explorer 11.'
+title: Internet Explorer 11 testing
+localization_priority: Normal
+---
+
+## Internet Explorer 11 testing
+
+Depending on the specifications of your add-in, you may plan to support older versions of Windows and Office, which require testing on Internet Explorer 11. This is often necessary as part of submitting your add-in to AppSource. You can use the following command line tooling to switch from more modern runtimes used by add-ins to the Internet Explorer 11 runtime for this testing.
+
+> ![TIP]
+> It isn't necessary to use this tooling, but it should help debug the majority of issues related to the Internet Explorer 11 runtime. For complete robustness, you should test using a computer with a copy of Windows 7 and Office 2013 installed.
+
+## Pre-requisites
+
+- [Node.js](https://nodejs.org/) (the latest [LTS](https://nodejs.org/about/releases) version)
+- A code editor. We recommend [Visual Studio Code](https://code.visualstudio.com/)
+- [Be part of the Office Insider program](https://insider.office.com)
+
+These instructions assume you have set up a Yo Office generator project before. If you haven't done this before, consider reading a quick start, such as [this one for Excel add-ins](../quickstarts/excel-quickstart-jquery.md).
+
+## Using IE11 tooling
+
+1. Create a Yo Office generator project. It doesn't matter what kind of project you select, this tooling will work with all project types.
```

Comment:
I think we should we add a note saying that people can install the package directly from npm if they already have an existing project.